recipes/samurai/all/test_package/CMakeLists.txt

```diff
@@ -6,5 +6,3 @@ find_package(samurai REQUIRED CONFIG)
 add_executable(${PROJECT_NAME} test_package.cpp)
 
 target_link_libraries(${PROJECT_NAME} PRIVATE samurai::samurai)
-
-target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
```

Comment:
It seems the removal of this line could be the one causing the issues? I'd suggest leaving it.

Reply:
OMG you're right. I was thinking about the new release and I forgot that the older ones are also compiled.